jcl/src/openj9.sharedclasses/share/classes/com/ibm/oti/shared/SharedClassURLHelperImpl.java

```diff
@@ -47,12 +52,28 @@
 		initializeShareableClassloader(loader);
 	}
 	
-	private static native void init(); 
+	private static native void init();
+	
+	/* The passed-in URL path contains a jar file */
+	private boolean newJarFileCheck(URL convertedjarPath) {
+		boolean newJarFile = false;
+		String protocol = convertedjarPath.getProtocol();
```

Comment:
Please avoid doing anything extra when minimizeUpdateChecks is true. There is no point in doing extra computation to determine a new jar file when the native code will just ignore it.

Reply:
Already updated as suggested.